### PR TITLE
Activity Page Updates

### PR DESCRIPTION
**Pull request by jakekrog**
This branch makes the following changes to the Activity page:
- Update list layout per HIG
- Add completion message once last daily activity is completed

### CODE REVIEW

File: APCAppCore/APCAppCore/Library/AppearanceHelpers/APCAppearanceInfo.m [hunk 1]
```diff
@@ -72,7 +72,8 @@ + (NSDictionary *)defaultAppearanceDictionary
              kTertiaryPurpleColorKey : [UIColor colorWithRed:0.574 green:0.252 blue:0.829 alpha:1.000],
              kTertiaryGrayColorKey : [UIColor colorWithRed:157/255.0f green:157/255.0f blue:157/255.0f alpha:1.000],
              kBorderLineColor: [UIColor colorWithWhite:0.749 alpha:1.000],
-             kPrimaryNavBarColorKey : [UIColor whiteColor]
+             kPrimaryNavBarColorKey : [UIColor whiteColor],
+             kConfirmationCheckmarkColorKey : [UIColor colorWithRed:0.39 green:0.76 blue:0.46 alpha:1]
```
Comment (syoung-smallwisdom) [inline APCAppCore/APCAppCore/Library/AppearanceHelpers/APCAppearanceInfo.m:76]: Please do not use inline hardcoded colors

File: APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m [hunk 2]
```diff
@@ -107,12 +112,19 @@ - (void) viewDidLoad
 
     // make sure we know the state of CoreMotion permissions so it's available when we need it
     [self.permissionManager requestForPermissionForType:kAPCSignUpPermissionsTypeCoremotion withCompletion:nil];
+
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```
Comment (syoung-smallwisdom) [inline APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m]: Please remove this. This is shared code between different apps. A default should be NO with the option for a given app to override and set to YES.

File: APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m [hunk 3]
```diff
@@ -493,16 +528,31 @@ - (void) fetchNewestSurveysAndTasksFromServer: (id) __unused sender
...
-    
 }
 
-- (void) dismiss
+- (void) dismissSpinnerController
 {
-    [self dismissViewControllerAnimated:YES completion:nil];
+    [self dismissViewControllerAnimated:YES completion:^{
+        if (shouldShowTodayActivitiesCompleteModal && self.countOfRemainingTasksToday == 0 && previousCountOfRemainingTasksToday == 1) {
+            [self showActivitiesCompleteModal];
```
Comment (syoung-smallwisdom) [inline APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m]: Not keen on this UX flow. Added a message cell to the daily activities section that is displayed once all the activities are completed is a nice UX addition. Adding more buttons that the user needs to push - not so much.

**Comment by syoung-smallwisdom** [comment]
I believe this would require going to the IRB. With that in mind, it might be interesting as an optional include but we cannot accept this as is. Sorry.

**Comment by evliu** [comment]
Hi @syoung-smallwisdom , the newest commit removes the completion message modal. Could you please clarify not using an inline hardcoded color in APCAppearanceInfo.m? I tried to follow the pattern established with the rest of that file. Thanks!

File: APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m [hunk 4]
```diff
@@ -35,28 +35,29 @@
...
 
 static CGFloat const kTintedCellHeight             = 65;
 static CGFloat const kTableViewSectionHeaderHeight = 77;
-
+static CGFloat const kActivityHeaderTitleLabelFontSize = 20.f;
+static CGFloat const kActivityHeaderSubTitleLabelFontSize = 12.f;
+static NSString *const kActivityHeaderBackgroundImage = @"Activity_Header_Background";
+static NSString *const kAPCSectionTitleKeepGoing = @"Additional Activities";
+static NSString *const kAPCSectionSubtitleKeepGoing = @"Try one of these extra activities to enhance your study experience.";
+static NSString *const kAPCSectionSubtitleYesterday = @"Below are your incomplete tasks from yesterday. For your reference only.";
```
Comment (syoung-smallwisdom) [inline APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m]: static strings. need to be localized and if you are using as static strings then need to listen for language change and change the language accordingly.

**Comment by syoung-smallwisdom** [comment]
closing

**Pull request closed without merging**